### ISSUE TITLE
SVHN dataset

### ISSUE THREAD

**Issue by omg777** [GakkiAogSan/DANN#1]
Hello, Thanks for your nice work!

Where can I get the SVHN and MNIST dataset you used.

**Comment by omg777** [comment]
@GakkiAogSan 
+ Why did you use target_train_loader to test ?

**Comment by hehai131** [comment]
> Hello, Thanks for your nice work!
> 
> Where can I get the SVHN and MNIST dataset you used.

Sorry for my negligence, the svhn and mnist is on http://www.cs.umd.edu/~yogesh/datasets/digits.zip. As for mnist_m, maybe you can download from https://drive.google.com/open?id=0B_tExHiYS-0veklUZHFYT19KYjg.

**Comment by hehai131** [comment]
> @GakkiAogSan
> 
> * Why did you use target_train_loader to test ?

I don't know if I understand it wrong, there are some method use the target_train data for test. Maybe in unsupervised domain adaptation, target train data have no label, that can be considered as test data simultaneously.
